src/client/pages/Download/components/AppImage.tsx

```diff
@@ -6,33 +6,50 @@ type Props = {
   width: number
 }
 
-const Image = styled.img<Props>`
-  width: ${({ width }) => width}px;
+const Image = styled.img<{ imageWidth: number }>`
+  width: ${({ imageWidth }) => imageWidth}px;
```

Comment:
Question since I truly don't know this: will setting the dimensions directly as attributes on the `img` have any effect on layout shift (or anything else) when the dimensions are also, already, set with css like this?

Reply:
We are only setting the width of the image so there's not enough information to calculate the aspect ratio.

I think it's necessary to set width/height directly on the img element to enable automatic calculation of `aspect-ratio` - see here: https://web.dev/optimize-cls/

Reply:
That's very true 🧠  What I guess I'm worried about then is that we're saying that these images will be **x** size but then when the CSS does its thing the size is actually **y** - aren't we still at the risk of causing a layout shift then?

Reply:
CSS is evaluated before anything is render on screen though (https://css-tricks.com/browser-painting-and-considerations-for-web-performance/).

So the browser will have both the HTML and CSS information available. The only thing is doesn't know on first render/paint is the image data which is not yet loaded.

So browser will use both HTML and CSS to calculate size of the image and avoid layout shift. At least I'm fairly sure this is how it works.

Reply:
Ah, OK cool! Thank you for your explanation (and patience) 👍